studio/http_artifact_store.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import requests
+
+from tartifact_store import TartifactStore
+logging.basicConfig()
+
+
+class HTTPArtifactStore(TartifactStore):
```

Comment:
unlike other *ArtifactStore classes, this class only communicates to one artifact with a specific (signed) url, so it has to be instantiated for every artifact.